### DISCUSSION TITLE
New Pattern for a Tailor's ham/seam roll

### DISCUSSION THREAD

**Discussion by darigovresearch** [freesewing/freesewing#5774] [Design request]
Idea from Benjamin F. on the 6th of Jan 2024  community call

https://github.com/freesewing/freesewing/discussions/5532

**Comment by BenJamesBen** [comment]
- The tailor's ham would be simple, made from two oval-ish pieces. No fancy pattern with darts.
- The default size would be identical to the Dritz tailor's pressing ham. https://www.joann.com/tailor-s-ham/8590259.html
- There would be an inner bag to hold the filling, along with an outer cover.
- A design option might be for a more pointed end for 1950s fashion.
- The default tailor's pressing seam roll size would be identical to the Dritz seam roll. https://www.joann.com/dritz-seam-roll-for-pressing-seams/8590234.html
- A possibility for an additional item might be a contour (tear-drop shaped) tailor's pressing ham.

For details about proper construction:
- https://www.youtube.com/watch?v=5S9KpMRDG_s
- https://www.youtube.com/watch?v=khHWUwm24CY

**Comment by BenJamesBen** [comment]
Closing because there does not seem to be any interest for active discussion about this topic, per the April 27, 2024 Community Call. https://github.com/freesewing/freesewing/discussions/5724#discussioncomment-9247197

(Please feel welcome to re-open this Discussion if there is interest in resuming active discussion about the topic.)
